Sources/Datadog/Versioning.swift

```diff
@@ -1,3 +1,3 @@
 // GENERATED FILE: Do not edit directly
 
-internal let sdkVersion = "1.8.0-beta1"
+internal let __sdkVersion = "1.8.0-beta1"
```

Comment:
alternatively that could be `defaultSDKVersion` too

Reply:
My reasoning was: this value should **be never used** directly in our code - we should always use resolved `configuration.sdkVersion`. 

To avoid potential mistakes, I came up with this underscore notation `__sdkVersion`, hoping it will raise a CR flag when used in the code. The `defaultSDKVersion` looks like any other code we write, hence it could be misused and overlooked. 

If you think relying on the `__` prefix is too weak protection, I can add a linter rule for preventing direct usage of `__sdkVersion` in other files (like [we do with `UIApplication.shared`](https://github.com/DataDog/dd-sdk-ios/blob/6b735fb9ac5d704fd732842a59d5e7602870768d/tools/lint/sources.swiftlint.yml#L81-L89)).